### DISCUSSION TITLE
Add support for Notification Stacks

### DISCUSSION THREAD

**Discussion by flowgunso** [iSoron/uhabits#243] [Feature Requests]
The notification drawer can get filled with habit's reminders notifications. How about supporting stacked notifications ?

There's [documentation](https://developer.android.com/training/wearables/notifications/stacks.html) on Android Notifications stacks.

**Comment by iSoron** [comment]
Yes, this is something we should implement. Thank you for the suggestion.

**Comment by iSoron** [comment]
This feature has been added to the app, but I will try it for a while before deciding whether to keep it or not. On one hand, the notifications do look nicer bundled together, but, on the other hand, it's now harder to check the habits directly from the notification panel; you must first expand the group and then expand the notifications for the actions to show up. I am not sure most users even know how to do this.

If anyone has any thought on this, please let us know.

**Comment by 4ut0-M-4t** [comment]
@iSoron, full notification drawer is the biggest cause of frustration really. You can add another **Reminder** setting defaulting to `OFF`, like it is done in [Tasks](https://github.com/tasks/tasks), for those uncapable of dealing with stackable notifications on their smart devices. 
Please reconsider.
